### DISCUSSION TITLE
Performance test

### DISCUSSION THREAD

**Discussion by wangweiwei1188** [alibaba/xquic#138] [General]
Now, I do the performance test according to the test_server and test_client, but some problems encountered.

test_client: ret = xqc_stream_send(stream, user_stream->send_body, user_stream->send_body_len, 0);
test_server: read = xqc_stream_recv(stream, buff, buff_size, &fin);
test_server<------------------------------test_client

But it's exited according to timeout when send one user data from test_client to test_server.

**Comment by zhangqixuan0118** [comment]
Did you fix this? I'm having the same issue too
